driver/src/test/java/org/kaazing/k3po/driver/internal/behavior/visitor/InjectHttpStreamsVisitorTest.java

```diff
@@ -1,243 +0,0 @@
-            .done()
-        .done();
-        // @formatter:on
-
-        InjectHttpStreamsVisitor injectEvents = new InjectHttpStreamsVisitor();
-        inputScript.accept(injectEvents, new InjectHttpStreamsVisitor.State());
-    }
-
-    @Test
-    public void shouldAllowContentWithContentLength() throws Exception {
-        // @formatter:off
-        AstScriptNode inputScript = new AstScriptNodeBuilder()
-            .addAcceptStream()
-                .setLocation(URI.create("http://localhost:8000/somepath"))
-                .addAcceptedStream()
-                    .addReadConfigEvent()
-                         .setType("method")
-                         .setValueExactText("method", "get")
-                    .done()
-                    .addReadCloseCommand()
-                    .done()
-                    .addWriteConfigCommand()
-                        .setType("content-length")
-                    .done()
-                    .addWriteCommand()
-                        .addExactText("Some Content")
-                    .done()
-                    .addWriteCloseCommand()
-                    .done()
-                .done()
-            .done()
-        .done();
-        // @formatter:on
-
-        InjectHttpStreamsVisitor injectEvents = new InjectHttpStreamsVisitor();
-        inputScript.accept(injectEvents, new InjectHttpStreamsVisitor.State());
-    }
-
-    @Test
-    public void shouldAllowWriteAfterRequestResponseSwitchingProtocols() throws Exception {
-        // @formatter:off
-        AstScriptNode inputScript = new AstScriptNodeBuilder()
-            .addAcceptStream()
-                .setLocation(URI.create("http://localhost:8000/somepath"))
-                .addAcceptedStream()
-                    .addReadConfigEvent()
-                         .setType("method")
-                         .setValueExactText("method", "GET")
-                    .done()
-                    .addReadConfigEvent()
-                         .setType("header")
-                         .setValueExactText("name", "Upgrade")
-                         .addMatcherExactText("websocket")
-                    .done()
-                    .addWriteConfigCommand()
-                        .setType("status")
-                        .setValue("code", "101")
-                        .setValue("reason", "Switching Protocols")
-                    .done()
-                    .addWriteCommand()
-                        .addExactText("some websocket data")
-                    .done()
-                    .addCloseCommand()
-                    .done()
-                    .addClosedEvent()
-                    .done()
-                .done()
-            .done()
-        .done();
-        // @formatter:on
-
-        InjectHttpStreamsVisitor injectEvents = new InjectHttpStreamsVisitor();
-        inputScript.accept(injectEvents, new InjectHttpStreamsVisitor.State());
-    }
-
-    @Test(
-            expected = IllegalStateException.class)
```

Comment:
Are there equivalent tests to ensure the grammar or something else is now catching these?  I believe if someone writes a test as written below it is invalid, and should throw some sort of parse exception.  So my expectation is we need these tests or an equivalent